docs/cosa/run.md

```diff
@@ -257,6 +257,40 @@ $ cosa run -c --netboot tmp/pxelinux/pxelinux.0 -m 4096
 
 See [this section](https://docs.fedoraproject.org/en-US/fedora-coreos/live-booting/#_booting_via_pxe) of the official docs for more info.
 
+### PXE with Shim and UEFI Secure Boot
```

Comment:
This is technically GRUB, right? So maybe let's move this to after the `### GRUB` section and call it e.g. `### GRUB (Secure Boot)`?

Reply:
True